src/entities/drawer/ui/DrawingBoard.tsx

```diff
@@ -64,23 +62,23 @@ const DrawingBoard: FC<Props> = props => {
 
   const pathsCache = useRef<Array<SkPath>>([]).current;
 
-  const normalizedLines = useMemo(() => {
-    return transformByWidth(value, width);
-  }, [value, width]);
+  const currentLines = useMemo(() => {
+    return [...value];
+  }, [value]);
```

Comment:
I suppose we don't need to use useMemo here already in this case

Reply:
Then we'd get a new instance of array every render